api/v1alpha1/release_types_test.go

```diff
@@ -122,6 +138,31 @@ var _ = Describe("Release type", func() {
 		})
 	})
 
+	Context("When MarkDeployed method is called", func() {
+		It("should properly register the status if passed status is true or false", func() {
+			r.MarkDeployed(metav1.ConditionTrue, "CommitsSynced", "3 of 3 components deployed")
+			statusCondition := meta.FindStatusCondition(r.Status.Conditions, BindingDeploymentStatusConditionType)
+			Expect(statusCondition.Status).To(Equal(metav1.ConditionTrue))
```

Comment:
You could/should use MatchFields in here and in the other instances where you check the content of structs. You have an example in the `binding_test.go` file.